express-api/src/typeorm/Migrations/1716919102840-DefaultSortOrderIsDisabled.ts

```diff
@@ -0,0 +1,273 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DefaultSortOrderIsDisabled1716919102840 implements MigrationInterface {
+  name = 'DefaultSortOrderIsDisabled1716919102840';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "building_evaluation" DROP COLUMN "Date"`);
```

Comment:
This causes the migration to fail for me (does not exist). I imagine it's meant to be lower case, but I also think that this column should already be dropped by migration `EvaluationYearIntInsteadofDate1711054015973` (line 15).

Reply:
Hmm. Perhaps I was missing that migration run somehow. Will remove from this PR and just drop manually for my local.